content/millennium/r4/financial/account.md

```diff
@@ -0,0 +1,147 @@
+
+### Account Relationships
+
+![Account Relationship Model](/images/account-relationship-model.png)
+
+### Account Types
+
+The account resource supports multiple account types which are defined below. 
+
+* financial-account - Top level patient account
+* guarantor-balance - Self pay balance account.
+* statement - Snapshot in time of a patient statement.
+
+
+The following fields are returned if valued:
+
+* [Account id]( https://hl7.org/fhir/r4/resource-definitions.html#Resource.id ){:target="_blank"} 
+* [Account Identifier](https://hl7.org/fhir/R4/account-definitions.html#Account.identifier){:target="_blank"} 
+* [Status](https://hl7.org/fhir/R4/account-definitions.html#Account.status){:target="_blank"} 
+* [Type](https://hl7.org/fhir/R4/account-definitions.html#Account.type){:target="_blank"}
+* [Subject](https://hl7.org/fhir/R4/account-definitions.html#Account.subject){:target="_blank"} 
+* [Service period](https://hl7.org/fhir/R4/account-definitions.html#Account.servicePeriod){:target="_blank"}
+* [Owner](https://hl7.org/fhir/R4/account-definitions.html#Account.owner){:target="_blank"} 
+* [Guarantor](https://hl7.org/fhir/R4/account-definitions.html#Account.guarantor){:target="_blank"} 
+* [Part of](https://hl7.org/fhir/R4/account-definitions.html#Account.partOf){:target="_blank"}
+* [Extensions including related parts, balance, and state](#extensions){:target="_blank"}
+
+
+## Terminology Bindings
+
+<%= terminology_table(:account, :r4) %>
+
+## Extensions
+
+* [Related Parts]
+* [Balance]
+* [State]
+
+### Custom Extensions
+
+All URLs for custom extensions are defined as `https://fhir-ehr.cerner.com/r4/StructureDefinition/{id}`
+
+ ID                         | Value\[x] Type | Description
+----------------------------|----------------|-------------------------------------------------------------------------------------------
+ `related-parts`            | [`reference`]       | A reference to other related Accounts.
+ `balance`                  | [`money`]           | Represents the account balance.
+ `state`                    | [`codeableConcept`] | The status of the Account within the billing or correspondence workflow.
+
+
+## Search
+
+Search for Accounts that meet supplied query parameters:
+
+    GET /Account?:parameters
+
+### Authorization Types
+
+<%= authorization_types(practitioner: true, patient: false, system: true) %>
+
+### Parameters
+
+ Name         | Required?                                         | Type          | Description
+--------------|---------------------------------------------------|-----------------------------------------------------------------------------------------------------------
+ `_id`        | This or any other required search parameter       | [`token`]     | The logical resource id associated with the resource
```

Comment:
it took me a few parses to understand what's entailed here

i would suggest changing to one of these two options:

1:
_id - This; patient, identifier, and type; or -guarantor and type
identifier - This, patient, and type; _id; or -guarantor and type
type - This, patient, and identifier; _id; or -guarantor and type
patient - This, identifier, and type; _id; or -guarantor and type
-guarantor - This and type; patient, identifier, and type; or _id

2:
all params - See notes

below table
Notes:
* You may search via:
  * `_id` alone
  * `patient`, `identifier`, and `type`
  * `-guarantor` and `type`

Reply:
https://github.com/cerner/fhir.cerner.com/pull/291/commits/1fd3fccdcbc14acbba047a902a80e96873ec8594#diff-41181ca01179d0e3c17a1c551a4999beR81-R86